certification/internal/utils/file/helper.go

```diff
@@ -83,3 +85,73 @@ func ArtifactPath(artifact string) string {
 	}
 	return filepath.Join(artifactDir, artifact)
 }
+
+// Untar takes a destination path and a reader; a tar reader loops over the tarfile
+// creating the file structure at 'dst' along the way, and writing any files
+func Untar(dst string, r io.Reader) error {
```

Comment:
IMO, it would be better to pass the path to the tar file as an arg, rather than io.Reader.